src/m365/spo/commands/group/group-member-add.ts

```diff
@@ -195,14 +211,17 @@ class SpoGroupMemberAddCommand extends SpoCommand {
 
     const validUserNames: string[] = [];
     const invalidUserNames: string[] = [];
-    const userIdentifiers: string = args.options.userName || args.options.email || args.options.userId!.toString();
+    const userIdentifiers: string = args.options.userName || args.options.email || args.options.aadGroupId! || args.options.aadGroupName! || args.options.userId!.toString();
 
     return Promise
       .all(userIdentifiers.split(',').map(async userIdentifier => {
         try {
           if (args.options.userId) {
             await this.spoUserGet(args.options, userIdentifier.trim(), logger, validUserNames);
           }
+          else if (args.options.aadGroupId || args.options.aadGroupName) {
+            await this.aadGroupGet(args.options, userIdentifier.trim(), logger, validUserNames);
+          }
           else {
             await this.aadUserGet(args.options, userIdentifier.trim(), logger, validUserNames);
```

Comment:
maybe we may move the `userIdentifier.trim()` outside to have this in single place 🤔